examples/3d-tiles/app.js

```diff
@@ -103,6 +103,14 @@ export default class App extends PureComponent {
       container: this._statsWidgetContainer
     });
 
+    this._tilesetStatsWidget = new StatsWidget(
+      {},
```

Comment:
It is a nit, but I really wanted this to be "one line" only... 

- Can we specify `null` as the first argument instead of `{}`? 
- I believe framesPerUpdate is already 1 by default so we can drop that line.

Then creating a `StatsWidget` should not need so many lines...